jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -135,3 +140,17 @@ def test_write_momentmap(cubeviz_helper, spectrum1d_cube, tmp_path):
     plugin._obj.filename = "fake_path/test_file.fits"
     with pytest.raises(ValueError, match="Invalid path"):
         plugin._obj.vue_save_as_fits()
+
+
+@pytest.mark.remote_data
+def test_momentmap_nirspec_prism(cubeviz_helper, tmp_path):
+    uri = "mast:jwst/product/jw02732-o003_t002_nirspec_prism-clear_s3d.fits"
+    download_path = str(tmp_path / Path(uri).name)
+    Observations.download_file(uri, local_path=download_path)
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        cubeviz_helper.load_data(download_path)
+    plugin = cubeviz_helper.plugins['Moment Maps']
+    plugin.calculate_moment()
+    assert isinstance(plugin._obj.moment.wcs, WCS)
```

Comment:
I confirmed locally that this new test fails without the patch like this:
```
E       AssertionError: assert False
E        +  where False = isinstance(None, WCS)
```

Feel free to confirm it for yourself by running this test in a Jdaviz version without this patch.

Reply:
is it worth checking to make sure the WCS is actually the original?  Or is there no concern that that could ever not the be case and checking it exists is sufficient?

Reply:
More added to test, see below.